Sources/Client/WebSocket/Provider/StarscreamWebSocketProvider.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  StarscreamWebSocketProvider.swift
+//  StreamChatClient
+//
+//  Created by Alexey Bukhtin on 30/04/2020.
+//  Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+import Starscream
+
+final class StarscreamWebSocketProvider: WebSocketProvider {
```

Comment:
I would really like if we could just do `extension Starscream.WebSocket: WebSocketProvider { }` and just implement the bridging functions in an extension, but I understand it's probably not easily possible 😞

Reply:
Great idea! I'll think about that